### PR TITLE
Support BTreeSet.

### PR DESCRIPTION
**Pull request by Neopallium**
Add support for BTreeSet to the UI by re-using the `Vector` component.

### CODE REVIEW

**Comment by jacogr** [comment]
Closes https://github.com/polkadot-js/apps/issues/2059

File: packages/react-params/src/Param/Vector.tsx [hunk 1]
```diff
@@ -22,7 +22,8 @@ function getParam ([{ name, type }]: ParamDef[], index: number): ParamDef {
 }
 
 export function getParams (inputParams: ParamDef[], prev: ParamDef[], max: number): ParamDef[] {
-  if (prev.length === max) {
+  // HACK: I don't know why `max` changes to `undefined`.
+  if (prev.length === max || isUndefined(max)) {
```
[the remark this answers is not in the document]
Comment (jacogr) [reply packages/react-params/src/Param/Vector.tsx:26]: Weird. Yes, would need some debugging here - but should be ok to go in as-is. (If there was a a chain/extrinsic to test against can try and debug when in - sounds like some bug somewhere that manifests here…)

**Comment by Neopallium** [comment]
One downside is that the UI will not dedup or sort the values, which will cause the RPC nodes to reject the transaction for failing the decode -> encode roundtrip test.

I don't think the UI can automate the dedup/sorting while the user is editing (would make using the UI difficult).

**Review by jacogr — APPROVED** [review]
Thank you.

The sorting is an issue - it is something that also exists elsewhere (would need to search - same issue), but happy to have it without since it already is a massive improvement to the "normal hex input".

**Pull request merged**